### ISSUE TITLE
[Feature] Add automated removal of unused Wine builds

### ISSUE THREAD

**Issue by Lalarian** [lutris/lutris#2405]
Hello!

Lutris is a fine tool, and by having it installed for several months and experimenting with many games, the number of my installed games has risen to about 20 and the number of installed wine builds has increased, also.

Now I wanted to remove all the Wine builds that are not being used by any of my starters, but it is very bothersome to go trough all starters, note the build being used in a list and then go to "Manage Runners" and there to "Wine Version Management" and compare which one is not being used at all.
It would be great if there was a button for automatically uninstalling all builds that are not in use by any starter!

Thank you!

:)

**Comment by LeXofLeviafan** [comment]
…If you have this, please allow the user to disable this feature via settings. It's quite handy to use Lutris for managing Wine versions when one wants to keep a number of versions at hand to quickly try them on a game that may require specific conditions to work (or even use them for non-gaming applications).

**Comment by Lalarian** [comment]
It's so great that you already did it! :)

Lex, as you say is what I asked for: A button that scans for unused builds and removes them if they are not in use. If you don't want this, then you don't use the button. I assume they implented the feature like this. so there is no need for disabling anything.

**Comment by LeXofLeviafan** [comment]
Yes, "automated" normally means something that works without user input (which is why the user should always be able to disable such features if he wants to). If that's not what you meant then it should be referred to as a button (or option, or command) to remove unused Wine versions in the title.
P.S. I did miss you mention a button in the description, but "a button for an automatic action" is kind of an oxymoron, really :-D

**Comment by legluondunet** [comment]
Obsolete
